From c8094f3c69d0934dd2e0637ec665012762adf0d6 Mon Sep 17 00:00:00 2001
From: Lei Xu <lei@apache.org>
Date: Mon, 19 Sep 2016 11:44:03 -0700
Subject: [PATCH 2018/2848]  HDFS-10870. Wrong dfs.namenode.acls.enabled
 default in HdfsPermissionsGuide.apt.vm. (John
 Zhuge via lei)

(cherry picked from commit 2010dd302632c379702882126b5cf16cd170fd61)

 Conflicts:
	hadoop-hdfs-project/hadoop-hdfs/CHANGES.txt

Change-Id: Ia9e0c147fcfdfa669df7f38ae8514426a7efd039
---
 .../src/site/apt/HdfsPermissionsGuide.apt.vm       |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/site/apt/HdfsPermissionsGuide.apt.vm b/hadoop-hdfs-project/hadoop-hdfs/src/site/apt/HdfsPermissionsGuide.apt.vm
index b065446..9ecc43b 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/site/apt/HdfsPermissionsGuide.apt.vm
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/site/apt/HdfsPermissionsGuide.apt.vm
@@ -431,7 +431,7 @@ HDFS Permissions Guide
        The administrators for the cluster specified as an ACL. This
        controls who can access the default servlets, etc. in the HDFS.
 
-     * <<<dfs.namenode.acls.enabled = true>>>
+     * <<<dfs.namenode.acls.enabled = false>>>
 
        Set to true to enable support for HDFS ACLs (Access Control Lists).  By
        default, ACLs are disabled.  When ACLs are disabled, the NameNode rejects
-- 
1.7.9.5

